Makefile

```diff
@@ -128,34 +56,18 @@ build: build-server build-nightly-server build-ad-server build-client \
 .PHONY: debug-vars
 debug-vars:
 	@echo OS_NAME: $(OS_NAME)
-	@echo OS_PREFIX: $(OS_PREFIX)
 	@echo TAG: $(TAG)
 	@echo NIGHTLY_TAG: $(NIGHTLY_TAG)
 	@echo SERVER_NAME: $(SERVER_NAME)
-	@echo SERVER_REPO_NAME: $(SERVER_REPO_NAME)
-	@echo NIGHTLY_SERVER_REPO_NAME: $(NIGHTLY_SERVER_REPO_NAME)
 	@echo NIGHTLY_SERVER_NAME: $(NIGHTLY_SERVER_NAME)
 	@echo AD_SERVER_NAME: $(AD_SERVER_NAME)
-	@echo AD_SERVER_REPO_NAME: $(AD_SERVER_REPO_NAME)
-	@echo NIGHTLY_AD_SERVER_NAME: $(NIGHTLY_AD_SERVER_NAME)
-	@echo NIGHTLY_AD_SERVER_NAME: $(NIGHTLY_AD_SERVER_NAME)
-	@echo NIGHTLY_AD_SERVER_REPO_NAME: $(NIGHTLY_AD_SERVER_REPO_NAME)
-	@echo CLIENT_NAME: $(CLIENT_NAME)
-	@echo CLIENT_REPO_NAME: $(CLIENT_REPO_NAME)
-	@echo NIGHTLY_CLIENT_NAME: $(NIGHTLY_CLIENT_NAME)
-	@echo NIGHTLY_CLIENT_REPO_NAME: $(NIGHTLY_CLIENT_REPO_NAME)
-	@echo TOOLBOX_NAME: $(TOOLBOX_NAME)
-	@echo TOOLBOX_REPO_NAME: $(TOOLBOX_REPO_NAME)
-	@echo NIGHTLY_TOOLBOX_NAME: $(NIGHTLY_TOOLBOX_NAME)
-	@echo NIGHTLY_TOOLBOX_REPO_NAME: $(NIGHTLY_TOOLBOX_REPO_NAME)
 
 	@echo BUILDFILE_SERVER: $(BUILDFILE_SERVER)
 	@echo BUILDFILE_AD_SERVER: $(BUILDFILE_AD_SERVER)
 	@echo BUILDFILE_NIGHTLY_AD_SERVER: $(BUILDFILE_iNIGHTLY_AD_SERVER)
 	@echo BUILDFILE_NIGHTLY_SERVER: $(BUILDFILE_NIGHTLY_SERVER)
 	@echo BUILDFILE_CLIENT: $(BUILDFILE_CLIENT)
 	@echo BUILDFILE_TOOLBOX: $(BUILDFILE_TOOLBOX)
-	@echo BUILDFILE_NIGHTLY_TOOLBOX: $(BUILDFILE_NIGHTLY_TOOLBOX)
```

Comment:
What was the rationale behind removing specific lines and not others?

Reply:
I tried to remove variables that were no longer useful/interesting. BUILDFILE_NIGHTLY_TOOLBOX was a special case as it was never used anywhere else in the file.  I expect to have missed some stuff in my cleanups, the Makefile was (and still is) pretty messy. In the future we may be able to drop this debug target entirely, but I am trying not to make this PR too big and drastic.